gem/Rakefile

```diff
@@ -90,7 +94,14 @@ task :python_copy do
   copy_files(platform_config[:python], meterpreter_dest)
 end
 
-task :win_prep => [:create_dir, :win_compile, :win_copy] do
+task :create_manifest do
+  all_meterpreter_data = ::Dir.glob(dest + '/**/*').select { |f| ::File.file?(f) }.sort
+  manifest = all_meterpreter_data.map { |f| "#{f}:#{hash_type}:#{::OpenSSL::Digest.new(hash_type, ::File.read(f))}" }
```

Comment:
```suggestion
  manifest = all_meterpreter_data.map { |f| [f, hash_type, ::OpenSSL::Digest.new(hash_type, ::File.binread(f))].join(':') }
```

Reply:
We don't want the hashes to be different on windows/linux - so prefer `binread` and `binwrite` 

https://github.com/rapid7/metasploit-framework/pull/16325